content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+         to the message are highlighted, as is the visibility setting and the post button.
+
+.. note::
+   If the user is already signed in to X/Twitter, when the :guilabel:`Share Now` icon is clicked, an
+   X/Twitter page loads in a new tab with the draft message in a pop-up. If the user is *not*
+   already signed in, a sign-in screen loads instead, prompting the user to sign in to X/Twitter.
+
+LinkedIn
+~~~~~~~~
+
+To share a job position on LinkedIn, click the :guilabel:`Share Now` button with the :guilabel:`in
+(LinkedIn logo)` inside it. A new tab loads in LinkedIn, with a link to the job position at the top.
+
+The job position can be shared either in a public post, or in a private message to an individual or
+group of individuals.
+
+Share in a post
+***************
+
+To share the job position in a public post, click the :guilabel:`✏️ (pencil) Share in a post` button
+in the lower-left corner of the screen. A :guilabel:`Create a post` window loads, with the job
+position link populated. Enter any other information to the post in the field.
+
+To change the audience that will see the post, click the :guilabel:`🌎 (globe) Anyone` drop-down,
+located directly beneath the account name. Click on the desired selection, then click the
+:guilabel:`Save` button on the far right side of the screen, beneath the last listed option.
+
+:guilabel:`Everyone` allows anyone on LinkedIn to see the post. :guilabel:`Connections only` will
+only share the post with people who are connected with the user, and will be hidden from everyone
+else. :guilabel:`Group` shares the post with a specific group the user is a member of. The post can
+only be shared with one group; posting to multiple groups at once is not available. If
+:guilabel:`Group` is selected, a page loads with a list of all the groups the user is currently a
+member of. Click on a group to select it, then click the :guilabel:`Save` button in the lower right
+corner.
+
+Next, select who can comment on the post. To change this setting, click on the :guilabel:`🗩 (speech
+bubble) Anyone` button located beneath the body of the post. A page loads with the various options.
+Click on one of the three options to select it. The three options are :guilabel:`Anyone`,
+:guilabel:`Connections only`, and :guilabel:`No one`. The default option is :guilabel:`Anyone`,
+which allows anyone with a LinkedIn account to comment on the post. :guilabel:`Connections only`
+will only allow people who are connected with the user to comment on the post. :guilabel:`No one`
+turns off commenting, not allowing anyone to post. After making a selection, click the
+:guilabel:`Save` button in the far lower right corner.
+
+Finally, once the post is edited and the settings have been selected, click :guilabel:`Post` to
+share the job position.
+
+.. image:: referrals/linkedin.png
+   :align: center
+   :alt: A linkedIn post to share a job. The job position is highlighted as is the post button.
+
```

Comment:
Good point- I thought because it was me, the author, it was ok! =D But very good point- I changed this. Also, because I was signed in/needed to have access to the apps- I didn't make dummy copies of those apps yet. But I think I might need to in the future for this.

Reply:
Can't blur with nimbus unless it's a premium account, so i put a white box over my name after Lara- I figured, a first name only should be ok =)